modules/tensor_mechanics/test/tests/shell/static/abaqus_plate_bending.i

```diff
@@ -0,0 +1,208 @@
+# Shell element verification test from Abaqus verification manual 1.3.13
```

Comment:
Does this happen to be a NAFEMS problem?  Many of them are. I'd rather reference the NAFEMS benchmark than another code's verification manual if possible.

Reply:
I currently don't have access to the NAFEMS benchmark problems. So, I don't know whether it is a NAFEMS problem.

Reply:
Maybe it's not.  Could we still keep the comment there pointing to the abaqus problem, but rename the input file to something other than abaqus_plate_bending? Maybe just plate_bending?